app/plugin_hooks_test.go

```diff
@@ -678,11 +663,7 @@ func TestUserWillLogIn_Blocked(t *testing.T) {
 	defer th.TearDown()
 
 	err := th.App.UpdatePassword(th.BasicUser, "hunter2")
-
-	if err != nil {
-		t.Errorf("Error updating user password: %s", err)
-	}
-
+	assert.Empty(t, err, "Error updating user password: %s", err)
```

Comment:
As we are using `assert.Nil` for `nil` errors in other tests, can you please update this to use it for consistency?

Reply:
Sure @mgdelacroix I'll update the pr